deploy/storage-quota.conf

```diff
@@ -0,0 +1,24 @@
+
+# This control all the namespaces under root
+[*]
+metrics       =       1,000,000
+logical_size  = 250,000,000,000
```

Comment:
Are commas required here or optional?

Reply:
optional, added for a quick and small improvement of readability.